openreview/journal/process/preferred_emails_process.py

```diff
@@ -0,0 +1,50 @@
+def process(client, invitation):
+
+    journal = openreview.journal.Journal()
+
+    print('Get profiles for all the reviewers')
+    reviewers = client.get_group(journal.get_reviewers_id()).members
+
+    print('Get profiles for all the action editors')
+    action_editors = client.get_group(journal.get_action_editors_id()).members
+
+    print('Get profiles for all the assigned reviewers and action editors')
+    groups = client.get_all_groups(prefix=journal.venue_id + '/Paper')
```

Comment:
so not `/Submission`?

Reply:
Journal is using `Paper` instead of `Submission` :(, I think I would need to change this at some point.